espnet/asr/pytorch_backend/recog.py

```diff
@@ -65,16 +66,22 @@ def recog_v2(args):
     else:
         lm = None
 
+    if args.ngram_model:
+        ngram = NgramFullScorer(args.ngram_model, train_args.char_list)
```

Comment:
Maybe you should import `kenlm` and its scorer module locally if the python binding is optional.

Reply:
maybe import it in init function of Ngrambase

Reply:
done